### DISCUSSION TITLE
Looking for simple issues to start contributing to SWT

### DISCUSSION THREAD

**Discussion by fedejeanne** [eclipse-platform/eclipse.platform.swt#1180] [Q&A]
Hello everyone,
I would like to start contributing to SWT but there are no _[good first issues](https://github.com/eclipse-platform/eclipse.platform.swt/labels/good%20first%20issue)_ at the moment.

Does anyone have a tip on where to start? Otherwise I will just look at failed tests and maybe start from there.

Thank you in advance!

EDIT: I haven't contributed to SWT yet (newbie)

**Comment by merks** [comment]
There are many helpwanted tagged issues but that doesn’t make them a good first issue. 

I seem never to have time for this issue 

https://github.com/eclipse-platform/eclipse.platform.swt/issues/501

I don’t want to upgrade to windows 11 because of that issue.  (And I can’t reproduce it on windows 10.)  So that’s my selfish self-centered suggestion.

**Reply by iloveeclipse** [reply]
Very good proposal, especially there seem to be a work around proposed but not contributed!

**Reply by fedejeanne** [reply]
This one rings a bell. I'll look into it, @merks 

@iloveeclipse you mean this workaround? https://github.com/eclipse-platform/eclipse.platform.swt/issues/501#issuecomment-1827874450

**Reply by iloveeclipse** [reply]
Yes, but I haven't looked in detail if there are other workarounds or if that ome has any negative side effects.
